### ISSUE TITLE
Ability to collect and visualize a video for RL experiments

### ISSUE THREAD

**Issue by SGevorg** [aimhubio/aim#335]
When running RL experiments, there is a huge value in watching agent's interaction with the environment via a video.

One way to implement this: Collect RGB frames and  compile them into video or slider with certain frequency for good RL debugging experience.

**TODO:** more details needed for this feature's implementation

**Comment by tesfaldet** [comment]
Would love to see this implemented

**Comment by SGevorg** [comment]
It is going to be Indeed, but hopefully not by us. 
This is a relatively low hanging fruit right now, but the version of Aim we are working on right now is going to allow the users to build it themselves with very few lines of python code. We should have an alpha soon.

**Comment by tesfaldet** [comment]
Yeah awesome, I'd be down to help implement it once the new version gets released.

**Comment by tesfaldet** [comment]
Bump for this. How difficult would it be to implement video support?

**Comment by iwishiwasaneagle** [comment]
This feature is the only thing holding me back from completely switching from Weights And Biases or Tensorboard.

**Comment by hzaskywalker** [comment]
Bump again. What's the progress on this?

**Comment by NathanYee** [comment]
Just started using aim and this is the one big feature that I miss from other frameworks.

**Comment by DmitriyBalmashnov** [comment]
I'm willing to put in the time to help realize this, however I'm fairly new to the project, any pointers on where to start? @SGevorg
